stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/DOM.java

```diff
@@ -338,6 +361,22 @@ public void onChildRemoved(Object parentElement, Object childElement) {
     public RGBAColor contentColor;
   }
 
+  private static class InspectNodeRequestedEvent {
```

Comment:
This is consistent with the other "events" in this file (look above for a big handful of them). Should we just go ahead and rename them all right now?